aion_pool/src/Miningcore/Program.cs

```diff
@@ -193,6 +193,10 @@ private static void ValidateConfig()
             {
                 if (!config.EnableInternalStratum.HasValue)
                     config.EnableInternalStratum = clusterConfig.Relays == null || clusterConfig.Relays.Length == 0;
+
+                if (!config.SignatureSwapProtocolUpgradeBlock.HasValue) {
+                    config.SignatureSwapProtocolUpgradeBlock = long.MaxValue;
```

Comment:
It's a decent short term solution, I feel in the long term we should consider that mining pools shouldn't be privy to forking behaviour on the chain, so we should probably be propagating the block reward from the kernel itself into the software.

Reply:
agree.